collector/collector_test.go

```diff
@@ -28,12 +28,14 @@ import (
 
 	"github.com/spacemonkeygo/monotime"
 	"github.com/stretchr/testify/assert"
-	"github.com/uber-go/zap"
 	"github.com/uber/arachne/config"
```

Comment:
They should still be separate from standard libs, e.g. "net", "sync" right? Please see next diff.